Source/PeleC.H

```diff
@@ -742,6 +742,10 @@ protected:
 
   static amrex::Vector<int> src_list;
 
+  static bool use_chem_mask;
```

Comment:
The best way to do this is actually a bit different. You would add the option(s) here and run the `Source/Params/mk_params.sh` script to make this all automatic. I will do the definition/declaration/parmparse for you.

Reply:
Can you also add some documentation about what these new input parameters mean for the user?